Sources/StreamChat/Controllers/EventsController/EventsController.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension _ChatClient {
+    func createEventsController() -> EventsController {
+        .init(notificationCenter: eventNotificationCenter)
+    }
```

Comment:
```suggestion
    func makeEventsController() -> EventsController {
        .init(notificationCenter: eventNotificationCenter)
    }
```

Swift Guidelines specify that for factories we should use `make` instead of `create`

Reply:
renamed to `eventsController` to follow the naming convention used for other controllers